wyvern/entities/request.py

```diff
@@ -9,8 +11,17 @@ class BaseWyvernRequest(WyvernDataModel):
     request_id: str
     include_events: Optional[bool] = False
 
+    _identifier: Identifier = PrivateAttr()
```

Comment:
we're adding these interfaces for BaseWyvernRequest so that it behaves exactly the same as WyvernEntity until we actually migrate BaseWyvernRequest to inherit WyvernEntity (which will be done in a separate PR.)